common/country_types/hfe_country_types_pirate.txt

```diff
@@ -173,45 +181,60 @@ hfe_pirate = {
 		category = country_base
 		produces = {
 			energy = 50
-			minerals = 20
-			food = 10
+			minerals = 50
+			food = 50
 			alloys = 10
-			consumer_goods = 10
+			consumer_goods = 20
 			physics_research = 10
 			society_research = 10
 			engineering_research = 10
 			influence = 2
-			unity = 5
+			unity = 1
 		}
 		produces = {
 			trigger = {
 				is_difficulty > 0
 			}
-			influence = 0.2
+			influence = 0.25
+      physics_research = 5
```

Comment:
Why it is not tab intended

Reply:
That was an accident. I apologize and will fix it when I get around to it.

Reply:
Okay, done